test/functions/blockchainFunctionsTest.js

```diff
@@ -57,3 +57,75 @@ describe('historicalBalance', () => {
     expect(() => san.historicalBalance_(slug, from, to, ethAddress)).to.throw('No data')
   })
 })
+
+describe('ethSpentOverTime_', () => {
+  it('returns eth spent over a given time course', () => {
+    sandbox.stub(san.ApiClient_.prototype, 'fetchEthSpentOverTime').returns(
+      {
+        ethSpentOverTime: [{
+          datetime: '2019-07-21T12:40:34.000000Z',
+          ethSpent: 113.6400629999956
+        },
+        {
+          datetime: '2019-07-22T12:40:34.000000Z',
+          ethSpent: 0
+        },
+        {
+          datetime: '2019-07-23T12:40:34.000000Z',
+          ethSpent: 0
+        },
+        {
+          datetime: '2019-07-24T12:40:34.000000Z',
+          ethSpent: 0
+        }
+        ]
+      }
+    )
+
+    const response = san.ethSpentOverTime_(slug, from, to)
+
+    expect(response).to.deep.eq(
+      [
+        [
```

Comment:
This also looks strange, can have a row per array, elements are just two.

Reply:
Don't we run a formatter on this code? I think we could use prettier, for example?